app/src/main/java/org/mozilla/fenix/components/metrics/Metrics.kt

```diff
@@ -252,7 +252,7 @@ sealed class Event {
         enum class Item {
             SETTINGS, LIBRARY, HELP, DESKTOP_VIEW_ON, DESKTOP_VIEW_OFF, FIND_IN_PAGE, NEW_TAB,
             NEW_PRIVATE_TAB, SHARE, REPORT_SITE_ISSUE, BACK, FORWARD, RELOAD, STOP, OPEN_IN_FENIX,
-            SAVE_TO_COLLECTION, ADD_TO_HOMESCREEN
+            SAVE_TO_COLLECTION, ADD_TO_HOMESCREEN, QUIT
```

Comment:
@boek how do new events get into the metrics.md file? Do you have to change more files, or does the documentation automatically get added when you just add an event?

Should these be updated in the metrics.md file as part of the PR? At a quick glance I also didn't see save_to_collection or add_to_homescreen, so I'm wondering if I'm looking in the wrong place.

Reply:
I think we need to manually add this to `metrics.md` for now. At least until the Glean auto-generate documentation PR is merged 

@ekager

Reply:
Added to `metrics.md` and `metrics.yaml`!

Reply:
Unrelated, @boek does this mean open_in_fenix, save_to_collection, and add_to_homescreen did not get data review, or need to be updated?

Reply:
@liuche that is correct
https://github.com/mozilla-mobile/fenix/pull/2131/files
and https://github.com/mozilla-mobile/fenix/pull/4925 didn't get data reviews. In the short term I could put myself as a codeowner of `metrics.kt` until we get a better process